GlobalBadges/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "GlobalBadges",
-    "version": "1.2.6",
+    "version": "1.2.6.1",
```

Comment:
Versions like that will not work with my Updater. Could you change it back to `1.2.6` or bump it to `1.2.7`